### DISCUSSION TITLE
2.1.1 很不幸地又新建不了文章了

### DISCUSSION THREAD

**Discussion by Hulenkius** [Qexo/Qexo#214] [Q&A] [answered]
我的 scaffolds/post.md 内容如下：
```
---
title: ''
top: false
math: false
hidden: false
theme: flip
widgets: null
date: {{ date }}
author: ''
cover: ''
thumbnail: ''
category: ''
password: ''
---

<!--more-->
```

问题页面截图：
![image](https://user-images.githubusercontent.com/32562298/188063938-ee177cb9-9b15-491f-a1a2-72993f7dfda5.png)

**Comment by am-abudu** [comment, marked as answer]
已修复 4b6304985cfb44b71018ea305d726100162ef374

**Comment by Hulenkius** [comment]
~~问题未解决。~~

经检查，为 `<!--more-->` 行出问题，经删除，已解决。
